modules/admin_manual/pages/configuration/user/user_auth_twofactor.adoc

```diff
@@ -42,11 +53,9 @@ Because the user has to opt in, see the xref:{latest-webui-version}@webui:classi
 
 === Second Factor is Inaccessible
 
-In case a user loses access to the second factor, e.g. by breaking or losing the phone with two-factor SMS/app verification, the user is locked out. To give the user access to the account again, an admin can temporarily disable the two-factor check _for that user_ via the
-xref:configuration/server/occ_command.adoc#two-factor-authentication[occ commands for Two-Factor Authentication]. 
-After the issue has been fixed, the admin can reenable two-factor authentication for that user.
+In case a user loses access to the second factor, e.g. by breaking or losing the phone with two-factor SMS/app verification, the user is locked out. To give the user access to the account again, an admin can temporarily disable the two-factor check _for that user_ via the xref:configuration/server/occ_command.adoc#two-factor-authentication[occ commands for Two-Factor Authentication]. After the issue has been fixed, the admin can reenable two-factor authentication for that user.
```

Comment:
```suggestion
In case a user loses access to the second factor, e.g. by breaking or losing the phone with two-factor SMS/app verification, the user is locked out. To give the user access to the account again, an admin can temporarily disable the two-factor check _for that user_ via the xref:configuration/server/occ_command.adoc#two-factor-authentication[occ commands for Two-Factor Authentication]. After the issue has been fixed, the admin can enable two-factor authentication for that user again.
```